HaikuPorter/Options.py

```diff
@@ -80,6 +80,9 @@ def parseOptions():
 					  action='store_true', dest='debug', default=False,
 					  help="show Python stack traces for fatal errors")
 
+	parser.add_option('--no-git-repo',
```

Comment:
Am 08.01.2016 8:37 nachm. schrieb "waddlesplash" <notifications@github.com>:
>
> In HaikuPorter/Options.py:
>
> > @@ -80,6 +80,9 @@ def parseOptions():
> >    action='store_true', dest='debug', default=False,
> >    help="show Python stack traces for fatal errors")
> >
> > + parser.add_option('--no-git-repo',
>
> Maybe it'd be best to invert this (--git-repo) and default it to True?

I thought this option would only be interesting for buildbots. Now what?